src/utils/dom.js

```diff
@@ -28,7 +28,8 @@ export function siblings (node) {
  * @return {Boolean}
  */
 export function exist (node) {
-  if (node && node instanceof window.HTMLElement) {
+  // We are usine duck-typing here because we can't use `instanceof` since if we're in an iframe the class instance will be different.
+  if (node && node.appendChild && node.isConnected) {
```

Comment:
I agree that this seems like a better fix. However, I think it would be possible to avoid duck-typing by using `ownerDocument.defaultView`. Mostly I'm adding the comment because it seems like `ownerDocument` and `defaultView` are oft overlooked features of the DOM.
```suggestion
  if (!node) return false
  
  const nodeWindow = node.ownerDocument?.defaultView
  if (nodeWindow && node instanceof nodeWindow.HTMLElement) {
```
I’d also probably just return the test `nodeWindow && node instanceof nodeWindow.HTMLElement` directly and eliminate the following returns but couldn’t do that in the scope of the suggestion.